### PR TITLE
JAT-55 Add Sass styles and update project references

### PR DESCRIPTION
**Pull request by selinali2010**
### Style related changes
- Convert existing `.css` files into `.scss` files
- Add scss variables based on selected colour palette
- Add default styles for buttons, `Slider`s, and `Switch`s

### Other changes
- Update all references to Electron React to be AQUA
- Update package description to be relevant
- Update `registry.ts` file to use `import` and not `require`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -185,8 +173,8 @@
     "webpack-merge": "^5.8.0"
   },
   "build": {
-    "productName": "ElectronReact",
-    "appId": "org.erb.ElectronReact",
+    "productName": "AQUA",
+    "appId": "org.erb.AQUA",
```
Comment (xenown) [inline package.json:177]: maybe "org.jaty.AQUA" is better?

File: src/main/registry.ts [hunk 2]
```diff
@@ -1,20 +1,19 @@
-import path = require('path');
+import path from 'path';
 import { app } from 'electron';
+import { promisified as regedit, setExternalVBSLocation } from 'regedit';
 
-const regedit = require('regedit').promisified;
-
-const regeditNonPromise = require('regedit');
+// TODO: test that the import change here works for packaged apps!
```
Comment (xenown) [inline src/main/registry.ts:5]: I think we tested this right?

File: src/renderer/Slider.tsx [hunk 3]
```diff
@@ -147,29 +150,41 @@ export default function Slider() {
...
       <input
         type="range"
         min={MIN}
         max={MAX}
         value={preAmpGain}
         onChange={handleRangeInput}
+        style={
+          // Set css variables for determining upper/lower track
+          {
+            '--min': MIN,
```
Comment (xenown) [inline src/renderer/Slider.tsx:170]: I guess theres no way to access the min, max and value attribute of the input element from the css side?

File: src/renderer/Switch.tsx [hunk 4]
```diff
@@ -1,44 +1,56 @@
...
+  const [isKeyDown, setIsKeyDown] = useState(false);
   useEffect(() => {
     if (!peaceError) {
       onLoad();
     }
   }, [onLoad, peaceError]);
 
+  const handleKeyDown = (e: KeyboardEvent) => {
+    if (e.code === 'Enter' && !isKeyDown) {
+      handleToggle();
```
Comment (xenown) [inline src/renderer/Switch.tsx:29]: Shouldn't we only toggle on a keyUp event?

Comment (selinali2010) [reply src/renderer/Switch.tsx:29]: Good call, will make the change

**Review by xenown — APPROVED** [review]

**Pull request merged**